src/Form/formatters.test.js

```diff
@@ -61,36 +61,45 @@ describe('Can strip phone formatting characters', () => {
 
 describe('prepareFormValuesForStorage()', () => {
     describe('handling credentials', () => {
-        it('rejects for username only', () => {
-            const values = prepareFormValuesForStorage({
-                credentials: { username: 'dax@example.com' },
-                // @ts-ignore
-                creditCards: {},
-                // @ts-ignore
-                identities: {},
-            });
+        it.skip('rejects for username only', () => {
```

Comment:
Why skip? this isn't a requirement anymore so we either remove the test or update it.

Reply:
Ah damn! No I wanted to change this, sorry about that.